examples/mobile/UIComponents/components/controllers/slider-level-bar.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../lib/tau/mobile/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page" id="slider-level-bar-demo">
+		<div class="ui-header">
+			<h1>
+				Slider (Level bar)
+			</h1>
+		</div>
+		<div class="ui-content">
+			<div>
```

Comment:
Slider is next to Level Bar, not below in example from Guide
Would it be possible to place it next to?

Reply:
I'll fix it ... size of text is not defined. I'll try to fit some size.